src/core/qgismobileapp.cpp

```diff
@@ -200,7 +200,7 @@ QgisMobileapp::QgisMobileapp( QgsApplication *app, QObject *parent )
   if ( PlatformUtilities::instance()->capabilities() & PlatformUtilities::AdjustBrightness )
   {
     mScreenDimmer = std::make_unique<ScreenDimmer>( app );
-    mScreenDimmer->setActive( settings.value( QStringLiteral( "dimBrightness" ), true ).toBool() );
+    mScreenDimmer->setTimeout( settings.value( QStringLiteral( "dimBrightness" ), 40 ).toInt() );
```

Comment:
We might need to change the variable name here, otherwise people with a preexisting bool value here will end up with a timeout of 1 sec :fearful:

Reply:
ah, good spot, the variable name is wrong